lib/components/Modal/Modal.tsx

```diff
@@ -16,7 +16,7 @@ export interface ModalProps {
 export const Modal = ({ open, onClose, content, progress, sidePanel, footer }: ModalProps) => {
   const { sm } = useMediaQueries();
 
-  const heightClasses = `ds-max-h-[calc(100vh-172px)] ds-min-h-[calc(100vh-344px)] sm:ds-max-h-[calc(100vh-320px)]`;
+  const heightClasses = `ds-max-h-[calc(100vh-172px)] ds-min-h-[calc(100vh-344px)] sm:ds-max-h-[calc(100vh-204px)]`;
```

Comment:
The magical number of **204px** =
```
  40px (gap on top of the screen)
+ 24px (content top padding) 
+ 24px (content bottom padding) 
+ 76px (footer height) 
+ 40px (gap on bottom of the screen)
```

The result is then the maximum height that can be used for the actual content of Modal.

Reply:
Maybe this information could be helpful in the future, I'd leave it as a comment in the code.

Reply:
I added that info as a comment.